torch/testing/_internal/common_methods_invocations.py

```diff
@@ -681,6 +681,47 @@ def sample_inputs_index_select(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+def sample_inputs_sort(op_info, device, dtype, requires_grad):
+    def apply_grad(t):
+        if dtype in floating_types_and(torch.float16, torch.bfloat16):
+            t.requires_grad_(requires_grad)
+
+    def small_3d_unique(dtype, device):
```

Comment:
Are there any sample inputs with non-unique values? Why or why not?

Reply:
This is the part derived from legacy testing. There are cases with binary strings and multiple +-inf/nan values.